Neos.Flow/Classes/Session/Session.php

```diff
@@ -287,7 +287,7 @@ public function start()
         if ($this->request === null) {
             $requestHandler = $this->bootstrap->getActiveRequestHandler();
             if (!$requestHandler instanceof HttpRequestHandlerInterface) {
-                throw new Exception\InvalidRequestHandlerException('Could not start a session because the currently active request handler (%s) is not an HTTP Request Handler.', 1364367520);
+                throw new Exception\InvalidRequestHandlerException(sprintf('Could not start a session because the currently active request handler (%s) is not an HTTP Request Handler.', gettype($requestHandler)), 1364367520);
```

Comment:
Is that related?

Reply:
Nope, I just fixed the missing sprintf when I stumbled over this "(%s)".